apps/core/src/modules/enrollments/handlers/listEnrollments.ts

```diff
@@ -0,0 +1,24 @@
+import type { FastifyReply, FastifyRequest } from 'fastify';
+import { EnrollmentModel } from '@ufabcnext/models';
+
+export async function listEnrollments(
+  request: FastifyRequest,
+  reply: FastifyReply,
+) {
+  const user = request.user;
+
+  if (!user?.ra) {
+    return [];
+  }
+
+  const findEnrollmentQuery = {
+    ra: user.ra,
+    conceito: { $in: ['A', 'B', 'C', 'D', 'O', 'F'] },
```

Comment:
change "conceito" to concept or  letter grade here.

Reply:
this is unavoidable, all the documents in our enrollments table use the word "conceito" for the grade field.